parser/lexer.go

```diff
@@ -314,8 +314,8 @@ func (s *Scanner) handleIdent(lval *yySymType) int {
 	if !strings.HasPrefix(str, "_") {
 		return identifier
 	}
-	cs, err := charset.GetCharsetInfo(str[1:])
-	if err != nil {
+	cs, _ := charset.GetCharsetInfo(str[1:])
+	if cs == nil {
```

Comment:
The second argument here is still an error. If you don't check for it, our linter will complain.

Reply:
The linter can be passed, the previous failed was caused by ci.

Reply:
OK. I'll trust your judgement :-)